pkg/wasm/imports.go

```diff
@@ -34,7 +37,22 @@ func (rt *jsRuntime) Circle(r float64)           { circle(r) }
 func (rt *jsRuntime) Width(w float64)            { width(w) }
 func (rt *jsRuntime) Color(s string)             { color(s) }
 func (rt *jsRuntime) Clear(color string)         { clear(color) }
-func (rt *jsRuntime) Yielder() evaluator.Yielder { return rt.yielder }
+func (rt *jsRuntime) Stroke(s string)            { stroke(s) }
+func (rt *jsRuntime) Fill(s string)              { fill(s) }
+func (rt *jsRuntime) Dash(segments []float64)    { dash(floatsToString(segments)) }
+func (rt *jsRuntime) Linecap(s string)           { linecap(s) }
+
+func floatsToString(floats []float64) string {
+	if len(floats) == 0 {
+		return ""
+	}
+	var sb strings.Builder
+	sb.WriteString(fmt.Sprintf("%f", floats[0]))
```

Comment:
you could just range over all floats and then return `sb.String()[1:]` to strip off the space rather than treating the first one specially here. just an idea

Reply:
ok done.